boolasint.py

```diff
@@ -3,6 +3,6 @@
 True -> 1
 False -> 0
 """
-a = 5
+a = input("Enter your number here") 
```

Comment:
No need for manual input here. The travis job gets stuck - https://travis-ci.org/github/brennerm/PyTricks/builds/737610318